README.md

```diff
@@ -1,4 +1,44 @@
-# CorsixTH Community Content
-Fan levels and campaigns
+# CorsixTH custom campaigns, levels and maps
 
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />All work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+
+## Directory
+
+### Campaigns
+
+| Name                   | Author |
+|------------------------|--------|
+| No campaigns submitted |        |
+
+### Levels
+
+| Name                                             | Author                                          |
+|--------------------------------------------------|-------------------------------------------------|
+| [Earlytown](Levels/Earlytown/)                   | Matroftt                                        |
+| [Grand Hospital](Levels/GrandHospital/) (remake) | Matroftt                                        |
+| [Heaven Town](Levels/HeavenTown/)                | Lloyd LeJack                                    |
+| [Old Hillside](Levels/OldHillside/)              | Raven "Edgewurth" Penfold                       |
+| [Sandbox](Levels/Sandbox/)                       | Edvin "Lego3" Linge \| Jannikergudingenreklamus |
+| [Tetris](Levels/Tetris/)                         | Matroftt                                        |
+
+## Installing
+
+### Campaigns
+
+- Download the zip file into the `Campaigns` directory inside your CorsixTH user directory and unzip in place.
+- Ensure that all the files are inside a single directory and only one level deep
+
+### Levels
+
+- Download the `.level` and `.map` files directly and save into the `Levels` directory
+
+### Finding the CorsixTH user directory
+
+| Operating System          | Location (typical)                                             |
+|---------------------------|----------------------------------------------------------------|
+| Linux / macOS             | `${XDG_CONFIG_HOME}/CorsixTH/` or `~/.config/CorsixTH/`        |
```

Comment:
```suggestion
| Linux / macOS             | `${XDG_CONFIG_HOME}/CorsixTH/`, usually `~/.config/CorsixTH/`        |
```

The `XDG_CONFIG_HOME` usually has the value `.config`.

Reply:
The finding user directory table should likely also be mirrored into a wiki page (heck, even https://github.com/CorsixTH/CorsixTH wiki needs it)